playwright/tests/integration/properties/input.spec.ts

```diff
@@ -66,3 +66,21 @@ test('cmsQuickfix', async ({ page }) => {
   await label.expectValue('Firstname');
   await label.expectInputValue("#{ivy.cms.co('/Labels/Firstname')}");
 });
+
+test('badges', async ({ page }) => {
```

Comment:
I more thought at a unit test. E.g. which renders a BadgeInput (with the given array) and value and spits out the badge. 
Because then you have also automatically tested this hierarchy of the regex priority.